src/vsr/grid_scrubber.zig

```diff
@@ -481,10 +481,10 @@ fn ManifestBlockIteratorType(comptime ManifestLog: type) type {
             manifest_log: *const ManifestLog,
         ) ?vsr.BlockReference {
             // Don't scrub the trailing `blocks_closed`; they are not yet flushed to disk.
-            const log_block_count =
-                manifest_log.log_block_addresses.count - manifest_log.blocks_closed;
+            const log_block_count: u32 =
+                @intCast(manifest_log.log_block_addresses.count - manifest_log.blocks_closed);
```

Comment:
This cast is justified because `ManifestLog.checkpoint_references` returns `u32`.

Introducing an `fn ManifestLog.block_count() u32` would have been a more _clean_ solution, but it is perhaps not worth it to add this API, given that the field is going to be publicly accessible anyway.